src/Symfony/Component/Yaml/Parser.php

```diff
@@ -439,7 +442,8 @@ private function doParse(string $value, int $flags)
                     $value = '';
 
                     foreach ($this->lines as $line) {
-                        if ('' !== ltrim($line) && '#' === ltrim($line)[0]) {
+                        $leftTrimmedLine = ltrim($line);
```

Comment:
Actually we could move the `$trimmedLine` declaration here and then use `$trimmedLine` instead of `$leftTrimmedLine`, right?